ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TutorialPageFragment.kt

```diff
@@ -50,7 +52,35 @@ internal class TutorialPageFragment() : Fragment(), TutorialCallbacks {
 
     private fun ViewDataBinding.setupCompose() {
         when (this) {
-            is TutorialTipsLearnBinding -> compose.setContent { TipsTutorialLayout() }
+            is TutorialTipsLearnBinding -> compose.setContent {
+                TipsTutorialLayout(
+                    nextPage = { findListener<TutorialCallbacks>()?.nextPage() },
+                    onTutorialAction = { findListener<TutorialCallbacks>()?.onTutorialAction(it) },
+                    Page.TIPS_LEARN,
```

Comment:
for this parameter just use the `page` property that is available on the fragment. same goes for the other compose bindings